src/registers/xcontrol.rs

```diff
@@ -0,0 +1,95 @@
+//! Access to various extended system registers
+use bitflags::bitflags;
+
+/// Extended feature enable mask register
+#[derive(Debug)]
+pub struct XCr0;
+
+bitflags! {
+    /// Configuration flags of the XCr0 register.
+    pub struct XCr0Flags: u64 {
+        /// Enables x87 FPU
+        const X87 = 1;
+        /// Enables 128-bit (legacy) SSE
+        /// Must be set to enable AVX and YMM
+        const SSE = 1<<1;
+        /// Enables 256-bit SSE
+        /// Must be set to enable AVX
+        const YMM = 1<<2;
```

Comment:
I think `AVX` would be a better name for this flag.

Reply:
No quite because it also enables the extended SSE instruction set etc.

Reply:
No, I think it's fine. It's a good idea to stick to the official naming.